README.md

```diff
@@ -1,4 +1,108 @@
-ios-base-swift
-=========
+# ios-base-swift
 
-iOS swift template application
+iOS swift template application.
+
+## Get started
+
+To setup project follow a few simple steps:
+ 1. [Download](#download).
+ 2. [Rename project](#rename-project).
+ 3. [Configure mogenerator](#configure-mogenerator).
+ 4. [Configure CocoaPods](#configure-cocoapods).
+ 5. [Configure Fabrick/Crashlytics](#configure-fabric-crashlytics).
+ 6. [Edit README.md](#edit-readme-md).
+ 7. [Commit and push to repository](#made-initial-commit-and-push-to-remote-repository).
+
+### Download
+* Download prject as ```zip``` file.
+
+### Rename project
+To rename project you should:
+
+#### Change project name:
+* In the Project Navigator on the left side, click twice slowly and the Project file name will be editable. Type the new name. A sheet will appear with a warning and will list all the items Xcode believes it should change.
+* Accept the changes.
+
+#### Remove mentions in code:
```

Comment:
It's just every file....)
Also than better remove this, because it mark where and when file was created. If we want to change it... Than i think we just not need this. 
Also with this mark you can check if file template taken from our base project or other place